packages/dataviews/src/view-actions.js

```diff
@@ -161,18 +153,20 @@ function FieldsVisibilityMenu( { view, onChangeView, fields } ) {
 							} );
 						} }
 					>
-						{ field.header }
-					</DropdownMenuItem>
+						<DropdownMenuItemLabel>
+							{ field.header }
+						</DropdownMenuItemLabel>
+					</DropdownMenuCheckboxItem>
 				);
 			} ) }
-		</DropdownSubMenu>
+		</DropdownMenu>
 	);
 }
 
 // This object is used to construct the sorting options per sortable field.
 const sortingItemsInfo = {
-	asc: { icon: arrowUp, label: __( 'Sort ascending' ) },
-	desc: { icon: arrowDown, label: __( 'Sort descending' ) },
+	asc: { label: __( 'Sort ascending' ) },
```

Comment:
That was the main thing that I'm not sure if that's what we want or it's some technical restriction. Do we want to remove the sorting icons?

Reply:
It's a design spec — radio and checkbox items shouldn't display a decorative icon, since they already have the check icon. See https://github.com/WordPress/gutenberg/issues/55933#issuecomment-1814621699 for more info